### PR TITLE
[MIG] Python 3

### PR DESCRIPTION
**Pull request by PierrickBrun**
I created this branch to make a windows-compatible package, but the most part of the work was to migrate everything to python3.

Thanks to @rousseldenis , telium and cups were already migrated

What works and has been tested:
 [x] escpos (using https://github.com/akretion/py-xml-escpos/tree/py3)
 [x] display_driver (using https://github.com/akretion/pyposdisplay/tree/py3)
 [x] cups (tested printing with zebra and dymo printers)
 [x] win32print (tested with zebra and dymo printers)

What needs to be done:
 [] telium (may work, I did not test)
 [] serial
 [] document how to build windows .exe and how to install it

### CODE REVIEW

**Comment by rousseldenis** [comment]
@PierrickBrun Did you see that now, plugins can be selected individually by config (as I see you did some commits to make the same):

https://github.com/akretion/pywebdriver/pull/49/commits/9704bdce5e90805c489c39cf7d4a1760250c5313

**Comment by PierrickBrun** [comment]
@rousseldenis Indeed ! I don't know why I did not think about it sooner..

I removed these commits to make the PR more readable

File: pywebdriver/plugins/win32print_driver.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            ):
+                messages.append(printer["pPrinterName"])
+            state = {
+                "status": "connected",
+                "messages": messages,
+            }
+            return state
+
+        def printData(self, printer, data, title="Pywebdriver", options=None):
+            if printer.startswith("zebra"):
```
Comment (sebastienbeau) [inline pywebdriver/plugins/win32print_driver.py]: Maybe better to use the raw option in options

Comment (sebastienbeau) [reply pywebdriver/plugins/win32print_driver.py]: will support also copies options see available options here : https://github.com/akretion/ak-odoo-incubator/blob/10.0/proxy_action/models/helper.py#L27

**Comment by alexis-via** [comment]
I ported pypostelium on python3 and it works fine with pywebdriver on py3.
I now have 1 customer in production on pywebdriver on Ubuntu 20.04 in py3.
Let's merge!

**Comment by alexis-via** [comment]
pyposdisplay : use master branch
pypostelium : use master branch

**Pull request merged**